exploration/2025/fs_pixels.md

```diff
@@ -0,0 +1,232 @@
+---
+jupyter:
+  jupytext:
+    formats: ipynb,md
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.16.1
+  kernelspec:
+    display_name: ds-aa-nga-flooding
+    language: python
+    name: ds-aa-nga-flooding
+---
+
+# Floodscan pixels
+
+<!-- markdownlint-disable MD013 -->
+
+Grab Floodscan pixels along Benue in Adamawa.
+
+```python
+%load_ext jupyter_black
+%load_ext autoreload
+%autoreload 2
+```
+
+```python
+import ocha_stratus as stratus
+import matplotlib.pyplot as plt
+import pandas as pd
+import xarray as xr
+from tqdm.auto import tqdm
+from dask.diagnostics import ProgressBar
+
+from src.datasources import floodscan, hydrosheds
+from src.utils import blob
+```
+
+## Load data
+
+### Benue river
+
+```python
+benue = hydrosheds.load_benue_aoi()
+```
+
+```python
+benue.to_crs(3857).plot()
+```
+
+```python
+benue_buffer = benue.to_crs(3857).buffer(10 * 1000).to_crs(4326)
+```
+
+```python
+benue_buffer.plot()
+```
+
+### Floodscan
+
+#### Old `.nc` file
```

Comment:
Ideally we'd move away from this old file... but can save that for when we have better/more performant standard data loading functions in `stratus`